001_basics.md

```diff
@@ -312,10 +310,10 @@ Higher-Kinded Types
 -------------------
 
 The "type of types" in Haskell is the language of kinds. Kinds are either an
-arrow (``* -> *``) or a star (``*``).
+arrow (``k -> k'``) or a star (``*``).
```

Comment:
Yes, I suppose the previous definition was not consistent with higher kinded types.